packages/deployments/contracts/contracts/diamond/facets/RoutersFacet.sol

```diff
@@ -86,7 +123,7 @@ contract RouterPermissionsManagerFacet is Modifiers {
   modifier onlyRouterOwner(address _router) {
     address owner = s.routerInfo.routerOwners[_router];
     if (!((owner == address(0) && msg.sender == _router) || owner == msg.sender))
```

Comment:
[https://github.com/connext/nxtp/blob/d79aff6b9f854c44772e588e0ffbef29d03e680c/packages/deployments/contracts/contracts/diamond/libraries/LibConnextStorage.sol#L196](url)

See above. `routerInfo` renamed to `routerPermissionInfo`